unlock-wordpress-plugin/assets/build/css/fullpostpage.css

```diff
@@ -0,0 +1 @@
+.unlock-header-icon{background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns=%27http://www.w3.org/2000/svg%27 fill=%27%23FF6771%27 height=%2720%27 width=%2720%27%3E%3Cpath fill-rule=%27evenodd%27 clip-rule=%27evenodd%27 d=%27M12.522 11.121a3.64 3.64 0 0 0 .032-.5l.001-1.724v-.432H7.577a3914.548 3914.548 0 0 0 0 2.225c.001 1.083.656 1.985 1.702 2.35 1.43.5 3.044-.454 3.243-1.919zm2.981-7.096v.178a860.597 860.597 0 0 1-.005 2.866h.825v1.446h-.825c0 .725.003 1.449.01 2.173.022 2.295-1.451 4.179-3.416 4.902-2.075.763-4.02.466-5.74-.929-1.195-.967-1.839-2.25-1.852-3.796-.007-.783-.009-1.566-.01-2.35h-.815V7.07h.816l.001-.894.002-2.055c0-.018.002-.036.004-.056l.003-.035h3.05v3.04h4.892V4.025h3.06z%27/%3E%3C/svg%3E");background-repeat:no-repeat;background-size:20px;height:20px}.no-lock-address{background:red;color:#fff;font-size:15px;margin-top:5px;padding:5px 10px}.lock-warning{color:red;margin-top:-18px!important}.block-label{margin-bottom:2px}.block-label,.components-text.components-input-control__label{font-weight:600}.docs a{margin-bottom:10px;text-decoration:none}.setting-lock{border-bottom:1px solid #e0e0e0;margin-bottom:18px;padding-bottom:12px}.add-lock{border:1px solid var(--wp-admin-theme-color);color:var(--wp-admin-theme-color);margin:auto auto 16px}
```

Comment:
Was this auto-generated? If so let's not add it...

Reply:
anything in `build` should _not_ be added to `git`.

Reply:
This is confusing because the existing block and other implementations have their auto-generated files added to git. so wondering why this shouldn't be added for the new features I got added.

Anyways have included in the .gitignore file to henceforth ignore all files in the `build` folder and my latest PR does not include the full post/page auto-generated files.
So, consider this done too